Testscripts/Linux/ETHTOOL-MSG-LEVEL.sh

```diff
@@ -23,6 +23,37 @@ UtilsInit
 #######################################################################
 # Main script body
 #######################################################################
+# Check if feature is supported by the kernel
+if ! which nm; then
```

Comment:
move the check/install for nm in the if section below, only if netvsc is present as a module.